src/models/fill_in_the_blanks.rs

```diff
@@ -45,6 +45,11 @@ impl FillInTheBlanks {
     fn next(&mut self) {
         self.blank_index = self.blank_index % self.answers.len();
     }
+
+    pub fn instructions() -> String {
+        // TODO add instructions
+        return String::from("Press Q to quit")
```

Comment:
We should keep instructions consistent with the rest

```suggestion
        return String::from("q: Quit application")
```